template.js

```diff
@@ -0,0 +1,86 @@
+const getRequestHeader = require('getRequestHeader');
+const sendHttpRequest = require('sendHttpRequest');
+const JSON = require('JSON');
+const encodeUriComponent = require('encodeUriComponent');
+const Math = require('Math');
+const logToConsole = require('logToConsole');
+const getContainerVersion = require('getContainerVersion');
+
+
+const apiUrl = "https://api.openweathermap.org/data/2.5/weather?";
+const units = data.units;
+const apiKey = data.apiKey;
+const isLoggingEnabled = determinateIsLoggingEnabled();
+const traceId = isLoggingEnabled ? getRequestHeader('trace-id') : undefined;
+function getCity() {
+    const cityFromHeaders = getRequestHeader('X-Geo-City');
```

Comment:
There need to be ability to choose in UI from with parameter get City and Country
By default get from Stape headers  but provide ability to override this.